package.json

```
@@ -29,7 +29,7 @@
     "typescript": "4.6.2"
   },
   "peerDependencies": {
-    "@markdoc/markdoc": "^0.1.4",
+    "@markdoc/markdoc": "^0.2.0",
```

@kejiweixun let's just change this to 
```suggestion
    "@markdoc/markdoc": "*",
```